shared/src/business/entities/DocketEntry.ts

```diff
@@ -703,7 +715,7 @@ export class DocketEntry extends JoiValidationEntity {
   ): boolean => {
     if (!entry.isFileAttached) return false;
 
-    const petitionDocketEntry = getPetitionDocketEntry(rawCase);
+    const petitionDocketEntry = getPetitionDocketEntry(rawCase)!;
```

Comment:
This exclamation mark is a white lie because _technically_ we could return `undefined`. That said, throwing an error below this caused a whole lot of tests to fail because data wasn't mocked well, and I preferred to tell a white lie than spend hours fixing it.